pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -61,7 +63,9 @@ const (
 
 	// reconcilerSyncDuration is the amount of time the reconciler sync states loop
 	// wait between successive executions
-	reconcilerSyncDuration time.Duration = 5 * time.Second
+	// This has been increased from every 5 seconds to every 5 minutes since
+	// the timing has created an enormous amount of API traffic on such clouds as AWS.
+	reconcilerSyncDuration time.Duration = 5 * time.Minute
```

Comment:
@saad-ali that is the second PR that I wanted to drop.  Since I am not super familiar with wiring in the cobra options.